server/src/test/java/com/broadcom/lsp/cobol/usecases/engine/UseCaseEngine.java

```diff
@@ -216,6 +245,8 @@ private static void assertResultEquals(AnalysisResult actual, TestData expected)
     assertResult(
         "Section definition:", expected.getSectionDefinitions(), actual.getSectionDefinitions());
     assertResult("Section usages:", expected.getSectionUsages(), actual.getSectionUsages());
+
+    assertResult("Subroutine usage:", expected.getSubroutineUsages(), actual.getSubroutineUsages());
```

Comment:
Should assert also the subroutine definitions

Reply:
Done